packages/indexer-common/src/indexer-management/monitor.ts

```diff
@@ -80,7 +87,11 @@ export class NetworkMonitor {
       const result = await this.networkSubgraph.query(
         gql`
           query allocations($indexer: String!, $status: AllocationStatus!) {
-            allocations(where: { indexer: $indexer, status: $status }, first: 1000) {
+            allocations(
+              where: { indexer: $indexer, status: $status }
+              first: 1000
+              orderBy: createdAtBlockNumber
+            ) {
```

Comment:
Shall we be explicit here and specify the orderDirection?

Reply:
added `orderDirection: asc` for oldest ones first! Thanks for the suggestion